gm4_vecto_shamir/data/gm4_vecto_shamir/functions/vehicle/check_water.mcfunction

```diff
@@ -0,0 +1,13 @@
+#run from active
```

Comment:
nearly all of this function can be replaced with a predicate. The predicate can check for `fluid:"water"`. I think just checking a 3x3 for water or `#gm4_no_collision` will suffice.